doc/source/getting_started/installing.rst

```diff
@@ -0,0 +1,458 @@
+.. _installing_odl:
+
+##############
+Installing ODL
+##############
+
```

Comment:
A few words here or remove the "Introduction" title below. I find the style

Big Title
Small Title
text

To be quite hard to read and it feels like something was missed.